test/compliances/module-time-transfer-limits.test.ts

```diff
@@ -0,0 +1,526 @@
+import { loadFixture, time } from '@nomicfoundation/hardhat-network-helpers';
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { deployComplianceFixture } from '../fixtures/deploy-compliance.fixture';
+import { deploySuiteWithModularCompliancesFixture } from '../fixtures/deploy-full-suite.fixture';
+
+async function deployTimeTransferLimitsFixture() {
+  const context = await loadFixture(deployComplianceFixture);
+
+  const complianceModule = await ethers.deployContract('TimeTransfersLimitsModule');
+  await context.suite.compliance.addModule(complianceModule.address);
+
+  return {
+    ...context,
+    contracts: {
+      ...context.suite,
+      complianceModule,
+    },
+  };
+}
+
+async function deployTimeTransferLimitsFullSuite() {
+  const context = await loadFixture(deploySuiteWithModularCompliancesFixture);
+  const complianceModule = await ethers.deployContract('TimeTransfersLimitsModule');
+  await context.suite.compliance.addModule(complianceModule.address);
+
+  return {
+    ...context,
+    suite: {
+      ...context.suite,
+      complianceModule,
+    },
+  };
+}
+
+describe.only('Compliance Module: TimeTransferLimits', () => {
```

Comment:
Remaining ".only" to remove

```suggestion
describe('Compliance Module: TimeTransferLimits', () => {
```